account-compression/sdk/src/merkle-tree/index.ts

```diff
@@ -143,11 +172,22 @@ export class MerkleTree {
       }
       if (verbose) console.log(`node ${i} ${new PublicKey(node).toString()}`);
     }
+    return node;
+  }
+
+  static verify(
+    root: string,
+    merkleTreeProof: MerkleTreeProof,
+    verbose = false
```

Comment:
nit: for consistency how about
```suggestion
    verbose: boolean = false
```

Reply:
Fixed!